dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -704,20 +708,30 @@ private WorkflowVersion setupWorkflowFilesForGitHubVersion(Triple<String, Date,
                         testJson.setContent(testJsonContent);
 
                         // Only add test parameter file if it hasn't already been added
-                        boolean hasDuplicate = version.getSourceFiles().stream().anyMatch(
-                            (SourceFile sf) -> sf.getPath().equals(workflow.getDefaultTestParameterFilePath()) && sf.getType() == testJson.getType());
+                        boolean hasDuplicate = version.getSourceFiles().stream().anyMatch((SourceFile sf) -> sf.getPath().equals(workflow.getDefaultTestParameterFilePath()) && sf.getType() == testJson.getType());
                         if (!hasDuplicate) {
                             version.getSourceFiles().add(testJson);
                         }
+                    } else {
+                        missingParamFiles.add(testParameterPath);
                     }
                 }
+
+                if (missingParamFiles.size() > 0) {
+                    validationMessage.put("/.dockstore.yml", String.format("The following file(s) are missing: %s.",
+                            missingParamFiles.stream().map(paramFile -> String.format("'%s'", paramFile)).collect(Collectors.joining(", "))));
+                }
             }
         } else {
             // File not found or null
             LOG.info("Could not find file " + primaryDescriptorPath + " in repo " + repository);
-            return null;
+            validationMessage.put("/.dockstore.yml", "Could not find the primary descriptor file '" + primaryDescriptorPath + "'.");
```

Comment:
Can you make "/.dockstore.yml" a constant somewhere?  There was a recent ticket saying how it sometimes was "/dockstore.yml" and it seems like now is as good a time as any to just start using a constant for that file path.